src/containers/Masthead/MastheadContainer.tsx

```diff
@@ -179,7 +181,9 @@ const MastheadContainer = () => {
           />
           {config.feideEnabled && (
             <FeideLoginButton to="/minndla">
-              <FeideLoginLabel>{t('myNdla.myNDLA')}</FeideLoginLabel>
+              <FeideLoginLabel>
```

Comment:
Burde denne være med her også?
```suggestion
              <FeideLoginLabel data-hj-suppress>
```